dist/src/main/java/io/camunda/zeebe/shared/Profiles.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.shared;
+
+public class Profiles {
+  @SuppressWarnings("unused")
+  public static final String BROKER_STANDALONE = "broker & standalone";
+
+  public static final String BROKER_OR_GATEWAY_STANDALONE = "(broker | gateway) & standalone";
```

Comment:
:question: Couldn't we use meta annotations? For example:

```java
@Target({ElementType.METHOD})
@Retention(RetentionPolicy.RUNTIME)
@Documented
@Inherited
@Profile("broker & standalone")
public @interface ProfileStandaloneBroker {}
```

Then:

```java
@Component
@WebEndpoint(id = "backups")
@ProfileStandaloneBrokerProfile
public final class BackupEndpointStandalone {
```

I don't have a strong argument: it's clearer to me with the custom annotation, but it's likely a personal preference.

That said, Spring already provides a `Profiles` class, so I would at least rename it if we keep this approach.

Reply:
I originally put them as simple Strings so they can be easily tested with a standalone unit test. However I split "BackupEndpointStandaloneTest" with both profiles so I guess we are covering both cases anyway :shrug: 

If you see no values in the `ProfileTest` we can proceed with the meta annotations :+1:

Reply:
It's really just testing Spring - I understand the idea is to make sure if we ever change them something breaks (so encoding expectations/behavior), but it's likely still low value.

I didn't push it as a change request though because again, I think both approaches are fine - I don't have strong arguments for the meta annotation, just that it's easier to understand and maintain for me, but that's pretty weak :smile:

Reply:
Let's go with the meta annotation :+1: